### PR TITLE
Add prerequisites to README

### PR DESCRIPTION
**Pull request by alextu**
In particular, the Git CLI config instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,42 @@
 
 The Wrapper Upgrade Gradle Plugin creates tasks to upgrade the Gradle Wrapper for target projects hosted on GitHub.
 
+## Prerequisites
+
+To run the upgrade tasks, you'll need:
+
+* **Java 8 or later.**
+* **Git CLI.**  The plugin uses the Git CLI to commit, create branches, and push changes.
```
Comment (guylabs) [inline README.md:14]: Maybe add a link to the CLI, and I'd call it GitHub CLI.

**Review by guylabs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (StefMa) [reply README.md:14]: It's not the github cli, it's just "git" 🙃

Comment (guylabs) [reply README.md:14]: Then I'd just call it Git.

Comment (alextu) [reply README.md]: done

**Review by guylabs — APPROVED** [review]

**Pull request merged**